site/_includes/macros/card-authors.njk

```diff
@@ -7,18 +7,20 @@
     If there's no data for the author, just renders their ID.
   #}
   {% if authors.length < 3 %}
-    {% for item in authors %}
-      {% set author = authorsData[item] %}
+    {% for authorId in authors %}
+      {% set author = postsData.authors[authorId] %}
       {% if author.image %}
+      <a href="{{ author.url }}" class="card-authors__image">
         {% Img
-          class="card-authors__image rounded-full flex-shrink-none width-600 height-600 object-fit-cover",
+        class="flex-shrink-none width-600 height-600",
           src=author.image,
-          alt=author.name.given + author.name.family,
+          alt=author.title | i18n(locale),
```

Comment:
I appreciate your enthusiasm but I don't think we need to translate author names.

(I am a big fan of removing firstname/lastname though, thanks for that.)